source/test/E2E.test.js

```diff
@@ -1,82 +1,111 @@
+
+
+
+    },10000);
+
+    test('Check time button',async () => {
+        const puppeteer = require("puppeteer");
+        const browser = await puppeteer.launch();
+        const page = await browser.newPage();
+        await page.goto("https://big-bytez.github.io/cse110-fa21-group19/source/home-page/index.html",{
+            waitUntil: 'networkidle0',
+        });
+        // await page.goto("http://127.0.0.1:5500/source/home-page/index.html",{
+        //     waitUntil: 'networkidle0',//wait until all source loaded
+        // });
+
+        //wait untill page loaded
+
+        // click 10 mins button
+        await page.click("center button");
+        await page.waitForNetworkIdle();
+        // await page.screenshot({
+        //     path: "./search page1.png",
+        //     fullPage: true
+        // });
+
+        // await page.screenshot({
+        // path: "./search page2.png",
+        // fullPage: true
+        // });
 
-    //     const puppeteer = require('puppeteer');
-    //     const browser = await puppeteer.launch();
-    //     const page = await browser.newPage();
-    //     await page.goto('http://127.0.0.1:5500/source/home-page/index.html');
-    //     await page.waitForSelector('#tile');
+        
+        // Grab all Thumbnails
+        const numThumbnails = await page.$$eval('search-recipe', (prodItems) => {
+            return prodItems.length;
+        });
 
     //     let checkMark = true;
     //     setTimeout(async() => {await browser.close();},5000);
     //     const thumnail = await page.$(".container > a");
     //     await thumnail.click();
 
-    //     await page.waitForSelector("h1");
-    //     const title =  await page.$("h1");
-    //     let innertext = await title.getProperty("innerText");
+        await browser.close();
 
-    //     if (innertext['_remoteObject'].value == ""){
-    //         checkMark = false;
-    //     }
-        
-    //     await browser.close();
-    //     expect(true).toBe(true);
+        expect(numThumbnails > 0).toBe(true);
 
+    },20000);
 
-    // },5000);
+    test('Check search page',async () => {
+        const puppeteer = require("puppeteer");
+        const browser = await puppeteer.launch();
+        const page = await browser.newPage();
+        await page.goto("https://big-bytez.github.io/cse110-fa21-group19/source/home-page/index.html",{
+            waitUntil: 'networkidle0',
+        });
 
+        // await page.goto("http://127.0.0.1:5500/source/home-page/index.html",{
+        //     waitUntil: 'networkidle0',
+        // });
 
-    // it('Check user page routing', async () => {
 
-    //     const puppeteer = require('puppeteer');
-    //     const browser = await puppeteer.launch();
-    //     const page = await browser.newPage();
-    //     await page.goto('http://127.0.0.1:5500/source/home-page/index.html');
-    //     await page.waitForSelector('#tile');
+            await page.$eval('#query', el => el.value = "chicken");
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8454745)
```suggestion
            await page.$eval("#query", el => el.value = "chicken");
```